oryx-app-common/src/test/java/com/cloudera/oryx/app/kmeans/KMeansPMMLUtilsTest.java

```diff
@@ -62,54 +47,11 @@ public void testClustering() {
 
   @Test
   public void testPMMLToClusters() {
-    PMML pmml = buildDummyClusteringModel();
+    PMML pmml = KMeansPMMLUtils.buildDummyClusteringModel();
     List<ClusterInfo> clusterInfoList = KMeansPMMLUtils.read(pmml);
     assertEquals(NUM_CLUSTERS, clusterInfoList.size());
     assertEquals(NUM_DIMENSIONS, clusterInfoList.get(0).getCenter().length);
     assertEquals(CLUSTER_SIZE, clusterInfoList.get(1).getCount());
   }
 
-  public static PMML buildDummyClusteringModel() {
```

Comment:
Actually, this should go the other way. This is test-only code and I should never have put it in the non-test source directory. Are the two actually identical?